backend/middleware/__init__.py

```
@@ -0,0 +1,34 @@
+    "require_professor", 
+    "require_student",
+    "require_admin_or_professor",
+    "require_admin",
+    "get_user_role",
+    "is_professor",
+    "is_student", 
+    "is_admin",
+    "can_access_cohort",
+    "require_cohort_access",
+    "require_ownership_or_admin"
+]
```

Ruff (0.14.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)